### PR TITLE
Added explanations to readme

### PR DESCRIPTION
**Pull request by jensstigaard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,32 @@
 
 In this repository, you can find the lang files for the framework PHP, [Laravel 4/5/6/7](https://laravel.com).
 
-## New
+## News
 
-* in version 6.1, we propose a new file by language: `validation-inline.php` ( see #1268 )
-* in version 7, we propose new directory names to follow ISO-15897 ( see #1269 )
+* In Laravel version 6.1, we propose a new file by language: `validation-inline.php` ( see #1268 )
```
Comment (caouecs) [inline README.md:7]: No, it's the version 6.1 of this project, not Laravel :)

File: README.md [hunk 2]
```diff
@@ -2,23 +2,32 @@
...
+* in Laravel version 7, we propose new directory names to follow ISO-15897 ( see #1269 )
```
Comment (caouecs) [inline README.md:8]: the same, the version 7 of this project

Comment (jensstigaard) [reply README.md:8]: Why would you be interested in installing an old version of this project?

Comment (caouecs) [reply README.md:8]: This project is used by other projects to install easier the files. And some updates, like directory names, can break this projects.

Then, some versions are depending on Laravel version.

Comment (jensstigaard) [reply README.md:8]: Okay, so I revert these changes.

**Comment by caouecs** [comment]
Tak

**Comment by jensstigaard** [comment]
Je vous en prie (😄  ?)

**Pull request merged**